### PR TITLE
Onboarding

### PR DESCRIPTION
**Pull request by Robiquet**
[#609 ]

### CODE REVIEW

**Comment by Robiquet** [comment]
Will add to the new modal component on my other branch

File: components/account/OnboardingModal.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            src={wallet.logo.src}
+            alt={wallet.logo.alt}
+            width={30}
+            height={30}
+            quality={100}
+          />
+          <div className="relative font-medium text-ztg-18-150 ml-[15px]">
+            <span>{wallet.title}</span>
+            {wallet.title === "Talisman" && (
+              <span className="hidden md:inline absolute left-[90px] top-[4px] text-ztg-12-120 font-medium bg-green-light text-green py-[4px] px-[8px] rounded-md">
```
Comment (robhyrk) [inline components/account/OnboardingModal.tsx:105]: I was testing on my browser and `sm:inline` breakpoint works without disrupting the layout

File: components/account/OnboardingModal.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+const OnBoardingModal = () => {
+  const [step, setStep] = useState(0);
+  return (
+    <div
+      className="flex flex-col gap-y-[20px] justify-center items-center bg-white border 
+                border-black h-[438px] w-full max-w-[526px] p-[30px] rounded-ztg-10"
+    >
+      <div className="rounded-full w-[120px] h-[120px] mb-auto">
+        <Image
+          alt="AI Logan?"
```
Comment (robhyrk) [inline components/account/OnboardingModal.tsx:162]: My thoughts as well!

File: components/account/OnboardingModal.tsx [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+const walletsConfig = [
+  new TalismanWallet(),
+  new PolkadotjsWallet(),
+  new SubWallet(),
+];
+
+const WalletSelection = observer(() => {
+  const handleWalletSelect = async (wallet: BaseDotsamaWallet) => {
+    window.open(wallet.installUrl);
```
Comment (robhyrk) [inline components/account/OnboardingModal.tsx:84]: Is there a way to detect by browser and direct accordingly? I don't think it's a big issue, but I did notice that on Mozilla it goes to the Chrome extension instead.

**Review by robhyrk — APPROVED** [review]

File: components/account/OnboardingModal.tsx (hunk 3 above)
Comment (Robiquet) [reply components/account/OnboardingModal.tsx:84]: Detect what by browser?

**Pull request merged**